packages/auth-construct/src/construct.test.ts

```diff
@@ -630,75 +624,159 @@ void describe('Auth construct', () => {
     });
   });
 
-  void describe('storeOutput', () => {
+  void describe('storeOutput in synthesized template', () => {
     let app: App;
     let stack: Stack;
-    const storeOutputMock = mock.fn();
-    const stubBackendOutputStorageStrategy: BackendOutputStorageStrategy<BackendOutputEntry> =
-      {
-        addBackendOutputEntry: storeOutputMock,
-        appendToBackendOutputList: storeOutputMock,
-      };
-
     void beforeEach(() => {
       app = new App();
       stack = new Stack(app);
-      storeOutputMock.mock.resetCalls();
     });
 
-    void it('stores outputs in platform - minimum config', () => {
-      const authConstruct = new AmplifyAuth(stack, 'test', {
+    void it('outputs defaults for minimal email config', () => {
+      new AmplifyAuth(stack, 'test', {
         loginWith: {
           email: true,
         },
-        outputStorageStrategy: stubBackendOutputStorageStrategy,
       });
+      const template = Template.fromStack(stack);
```

Comment:
note: instead of stubbing the outputStorageStrategy, we are using Template.fromStack to generate the final output, and validating that instead